documental/models.py

```diff
@@ -0,0 +1,415 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from documental.utils import (
+    diretorio_mapas_uso_solo,
+    diretorio_ti
+)
+
+
+class Action(models.Model):
+    """Action model data for documental model."""
+
+    # id = models.IntegerField(
+    #     _('Primary key'),
+    #     unique=True,
+    #     primary_key=True,
+    # )
+
+    no_acao = models.CharField(
+        _('Action name'),
+        max_length=255,
+        unique=True,
+    )
+
+    dt_cadastro = models.DateTimeField(
+        _('Registration date'),
+        null=True,
+        blank=True,
+    )
+    class Meta:
+        """"Meta class for `documental.Action` model."""
+        app_label = 'documental'
+        verbose_name = 'Action'
+        verbose_name_plural = 'Actions'
+
+    
+class Usuario(models.Model):
```

Comment:
User model?

Reply:
I will say "Class User" with this name, because this table makes reference to CMR1 and there it is with that name.